### PR TITLE
Fix arm CI

### PR DESCRIPTION
**Pull request by kirklandsign**
We need to use the new selective build, not the full portable kernel for the arm backend example.

### CODE REVIEW

File: .github/workflows/pull.yml [hunk 1]
```diff
@@ -106,6 +106,26 @@ jobs:
...
+        # The generic Linux job chooses to use base env, not the one setup by the image
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+        PYTHON_EXECUTABLE=python bash .ci/scripts/setup-linux.sh cmake
+        source .ci/scripts/utils.sh
+        install_flatc_from_source
+        install_executorch
+        # Test selective build
+        source /opt/arm-sdk/setup_path.sh
+        PYTHON_EXECUTABLE=python bash examples/arm/run.sh /opt/arm-sdk buck2
```
Comment (digantdesai) [inline .github/workflows/pull.yml:127]: `+x` here?

File: examples/arm/ethos-u-setup/core_platform/patches/0008-Use-new-LIB_ET_OP_REGISTRATION-lib-path.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+diff --git a/applications/executorch_tests/CMakeLists.txt b/applications/executorch_tests/CMakeLists.txt
+index 5f70e8e..a07f22f 100644
+--- a/applications/executorch_tests/CMakeLists.txt
++++ b/applications/executorch_tests/CMakeLists.txt
+@@ -43,7 +43,7 @@ message("**********************")
+
+ set(LIB_ET_RUNTIME "${ET_BUILD_DIR_PATH}/libexecutorch.a")
+ set(LIB_ET_ETHOS "${ET_BUILD_DIR_PATH}/backends/arm/libexecutorch_delegate_ethos_u.a")
+-set(LIB_ET_OP_REGISTRATION "${ET_BUILD_DIR_PATH}/kernels/portable/libportable_ops_lib.a")
++set(LIB_ET_OP_REGISTRATION "${ET_BUILD_DIR_PATH}/examples/arm/libarm_ops_lib.a")
```
Comment (digantdesai) [inline examples/arm/ethos-u-setup/core_platform/patches/0008-Use-new-LIB_ET_OP_REGISTRATION-lib-path.patch:19]: not changing this?

**Review by digantdesai — APPROVED** [review]

File: examples/arm/ethos-u-setup/core_platform/patches/0008-Use-new-LIB_ET_OP_REGISTRATION-lib-path.patch (hunk 2 above)
Comment (kirklandsign) [reply examples/arm/ethos-u-setup/core_platform/patches/0008-Use-new-LIB_ET_OP_REGISTRATION-lib-path.patch:19]: Let's do it later. I have trouble with docker pulling this change

File: .github/workflows/pull.yml (hunk 1 above)
Comment (kirklandsign) [reply .github/workflows/pull.yml:127]: Next iteration?

**Pull request closed without merging**